safe_transaction_service/history/services/index_service.py

```diff
@@ -119,17 +119,18 @@ def get_indexing_status(self) -> IndexingStatus:
 
         # Indexing points to the next block to be indexed, we need the previous ones
         erc20_block_number = min(
-            self.get_erc20_721_current_indexing_block_number() - 1, current_block_number
+            max(self.get_erc20_721_current_indexing_block_number() - 1, 0),
```

Comment:
This is not really related with the PR, could you add this change to the description?

Reply:
Done